utils/esr_peer_info.cpp

```diff
@@ -394,8 +394,8 @@ namespace
     const std::string
     data_string_for_get_peers(const std::string &swarm_id)
     {
-        static const auto NODE_LIST_ABI = str_to_json(GET_NODE_LIST_ABI);
-        static const auto GET_PEERS_ADDRESS{NODE_LIST_ABI["signature"].asCString() + 2}; // 0x46e76d8b -> 46e76d8b
+        const auto NODE_LIST_ABI = str_to_json(GET_NODE_LIST_ABI);
```

Comment:
you were better as a static here... why don;t you add a json version to the consts instead of converting it